Source/Parser/wp_parser.y

```diff
@@ -40,6 +42,8 @@
 %left '<' '>'
 %nonassoc NEG UPLUS
 %right POW
+%right GEQ
+%right LEQ
```

Comment:
I suggest you do `%left '<' '>' GEQ LEQ` instead.

Technically, it is not wrong.  But it's unnatural.  `right` means `a <= b <= c` is evaluated as `a <= (b <= c))`.  Probably not what one expects.  (Admittedly, also not one should write in any case.)  Anyway, I think we should use `%left` for `GEQ` and `LEQ`, the same association property as `<` and `>`, and I think you should put them in the same line as `<` and `>`.  The order of these lines matters in the following expression,

     x <= a**2

or

    -x <= 5

The expected behaviors are probably `x <= (a**2)` and `(-x) <= 5`, not `(x<=a)**2` and `-(x<=5)`.